src/RequestBuilder/ItemPropertiesGetterRequestBuilder.php

```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\RequestBuilder;
+
+use Fig\Http\Message\RequestMethodInterface;
+use Lmc\Matej\Model\Request;
+
+class ItemPropertiesGetterRequestBuilder extends AbstractRequestBuilder
```

Comment:
Its a builder, which build request to get item properties (Its not building "getter request"), so why not just `ItemPropertiesGetRequestBuilder`? (But only if the other one is kept as `ItemPropertiesSetupRequestBuilder`)